lib/kuzzle/kuzzle.ts

```diff
@@ -606,8 +607,32 @@ class Kuzzle extends KuzzleEventEmitter {
 
     try {
       for (const [type, importMethod] of Object.entries(this.importTypes)) {
+        const importPayload = {};
+
+        switch (type) {
+          case 'fixtures':
+            _.set(importPayload, ['toSupport', 'fixtures'], toSupport.fixtures);
+            break;
+          case 'mappings':
+            _.set(importPayload, ['toSupport', 'mappings'], toSupport.mappings);
+            _.set(importPayload, ['toImport', 'mappings'], toImport.mappings);
+            break;
+          case 'permissions':
+            _.set(importPayload, ['toSupport', 'securities'], toSupport.fixtures);
+            _.set(importPayload, ['toImport', 'profiles'], toImport.profiles);
+            _.set(importPayload, ['toImport', 'roles'], toImport.roles);
+            _.set(importPayload, ['toImport', 'users'], toImport.users);
+            break;
+        }
+
+        const importPayloadHash = sha256(JSON.stringify(importPayload));
```

Comment:
You should use `stable-json-stringify` here to ensure the key order will be the same (even if with Node.js there is very little chances to have a key order inconsistency)